src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -489,24 +499,25 @@ def deployment_options(self):
         ssds = 0
         nvmes = 0
         res = DeploymentOptions()
-        devices = {}
 
         for inventory_host in orch.inventory.list(hosts=None, refresh=True):
             for device in inventory_host.devices.devices:
                 if device.available:
-                    devices[device.path] = device
                     if device.human_readable_type == 'hdd':
                         hdds += 1
-                    elif device.human_readable_type == 'ssd':
+                    else:
                         ssds += 1
-                    elif device.human_readable_type == 'nvme':
+                        # we still don't know how to infer nvmes
```

Comment:
What about creating a tracker issue and even pasting here the link to that (`# TODO:...`).

Reply:
@epuertat done